python/ql/src/semmle/python/Stmts.qll

```diff
@@ -90,8 +90,10 @@ class AugAssign extends AugAssign_ {
     /* syntax: Expr += Expr */
     override Expr getASubExpression() { result = this.getOperation() }
 
+    /** Gets the target of this augmented assignment statement. */
```

Comment:
I assume that would be `x` in `x += 1`?

Reply:
Yep. I've updated the comment to say this explicitly.